pkg/client/clientset/versioned/typed/channels/v1alpha1/fake/fake_bus.go

```diff
@@ -36,7 +37,7 @@ type FakeBuses struct {
 
 var busesResource = schema.GroupVersionResource{Group: "channels.knative.dev", Version: "v1alpha1", Resource: "buses"}
 
-var busesKind = schema.GroupVersionKind{Group: "channels.knative.dev", Version: "v1alpha1", Kind: "Bus"}
+var busesKind = schema.GroupVersionKind{Group: "channels.knative.dev", Version: "v1alpha1", Kind: system.KindBus}
```

Comment:
This file is generated. Did you make these edits by hand? If so, they will be automatically removed in the future.